### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mservillat**

### CODE REVIEW

**Review by lmichel — APPROVED** [review]

**Comment by olebole** [comment]
@lmichel could you, before merging, give others at least chance to review?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
-# doc-template
-document template
+# Provenance data model recommendation v1.0
```
Comment (olebole) [inline README.md:1]: I would not put the version number to the README. With the first new commit, this is no longer version 1.0, but some unofficial version in between. Then the version number is quite misleading.

Comment (mservillat) [reply README.md:1]: the version is the one of the recommendation, the only official one. The readme is a copy of the IVOA document page page, not much to review here...

Comment (olebole) [reply README.md:1]: see #3 ...

**Pull request merged**